### PR TITLE
`Development`: Add documentation for text exercises

### PR DESCRIPTION
**Pull request by maximiliansoelch**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/guidelines/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context

Currently, there is no documentation for text exercises.

### Description

This PR adds basic documentation for text exercises.

### Steps for Testing

Check out the new documentation here: https://artemis-platform--6651.org.readthedocs.build/en/6651/user/exercises/textual.html

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

### CODE REVIEW

**Review by MichaelOwenDyer — DISMISSED** [review]
Looks good to me 👍🏽

File: docs/user/exercises/textual.rst [hunk 1]
```diff
@@ -1,2 +1,221 @@
...
+    :local:
+    :depth: 2
+
+
+Overview
+--------
+
+Conducting a Text exercise consists of 3 steps:
+
+1. **Instructor prepares exercise:** Creates and configures the text exercise in Artemis.
```
Comment (tunargs) [inline docs/user/exercises/textual.rst:13]: ...Instuctor creates and....

File: docs/user/exercises/textual.rst [hunk 2]
```diff
@@ -1,2 +1,221 @@
...
+              :align: center
+
+The following screenshot illustrates the first section of the form. It consists of:
+
+- **Title**: Title of an exercise.
+- **Categories**: Category of an exercise.
+- **Difficulty**: Difficulty of an exercise. (No Level, Easy, Medium or Hard).
+- **Mode**: Solving mode of an exercise. *This cannot be changed afterward* (Individual or Team).
+- **Release Date**: Date after which the exercise is released to the students.
+- **Release Date**: Date after which students can start the exercise.
```
Comment (tunargs) [inline docs/user/exercises/textual.rst:44]: Should be "Start Date"

File: docs/user/exercises/textual.rst [hunk 3]
```diff
@@ -1,2 +1,221 @@
...
+    .. figure:: text/import-text-exercise.png
+              :align: center
+
+- An import modal will prompt up, where you will have the option to select and import previous text exercises from the list by clicking on the **Import** button.
+
+    .. figure:: text/import-text-exercise-modal.png
+              :align: center
+              :scale: 50
+
+- Once you import one of the exercises, you will then be redirected to a form that is similar to *Create new text exercise* form with all the fields filled from the imported exercise. You can now modify the fields as per your necessity to create a new Text Exercise.
```
Comment (tunargs) [inline docs/user/exercises/textual.rst:98]: remove "new" (2 places)

Comment (tunargs) [inline docs/user/exercises/textual.rst:98]: uncapitalize T and E

File: docs/user/exercises/textual.rst [hunk 4]
```diff
@@ -1,2 +1,221 @@
...
+- When the exercise is released students can work on the exercise.
+
+    .. figure:: text/text-exercise-card-student-view.png
+              :align: center
+
+- They can start the exercise by clicking the |start| button.
+
+- Once they start the exercise, they will now have the option to work on it in an online text editor by clicking on the |open-text-editor| button.
+
+- The screenshot below depicts the exercise interface for students. They can read the Problem Statement on the right and fill in their solution in the textbox on the left. To submit, you need to click on the |submit| button on the top right.
```
Comment (tunargs) [inline docs/user/exercises/textual.rst:124]: uncapitalize P and S

**Review by tunargs — COMMENTED** [review]
Overall, looks fine. I just added a few suggestions.

**Review by tunargs — APPROVED** [review]
Approved 👍

**Review by DominikRemo — COMMENTED** [review]
Looks good 🚀
However, on small screens two images are distorted:

Bildschirmfoto 2023-06-06 um 08 51 19
Bildschirmfoto 2023-06-06 um 08 51 09

Probably has something to do with :scale:

**Pull request merged**